src/sklearn_evaluation/plot/clustering.py

```diff
@@ -78,28 +87,47 @@ def elbow_curve(X, clf, n_clusters=None, n_jobs=1, show_cluster_time=True, ax=No
     ax: matplotlib Axes
         Axes containing the plot
 
+    Notes
+    -----
+    .. deprecated:: 0.8.7
```

Comment:
We probably want to give users a heads up of minor or 2,
Did you check the guide about deprecation warnings?

Reply:
change to 0.8.9, besides that looks good.

Reply:
[guide](https://github.com/ploomber/contributing/blob/main/documentation/README.md#new-features-and-breaking-api-changes)

Reply:
done

Reply:
the lin Ido shared is outdated, this is the guide for deprecations: https://ploomber-core.readthedocs.io/en/latest/deprecation.html#parameter-renamed 

> change to 0.8.9, besides that looks good.

the guide also includes an example of the docstring, we use `.. versionchanged::` for parameter renaming

> We probably want to give users a heads up of minor or 2,

it's actually a period of two major releases. so if we deprecate this in the next release (0.8.7), we'll remove it in 0.10